src/ContentScript/injectors/youtube.plugin.ts

```diff
@@ -1,89 +1,101 @@
 /* eslint-disable class-methods-use-this */
 /* eslint-disable no-unused-vars */
 // import { debounce } from 'lodash';
-import LikeButton from '../../sdk/button.iframe';
-import eventCenter from '../event-center/index';
+import { renderYouTubeButton } from './render';
 
 class YoutubePlugin {
   youtubeStyle!: HTMLElement;
 
-  failCounter!: number;
-
   insertStyle() {
     this.youtubeStyle = document.createElement('style');
     this.youtubeStyle.innerHTML = `
               #meta-contents {
                 display: flex;
                 flex-direction: row;
                 justify-content: space-between;
+                padding-top:40px;
               }
-              .button-container {
+              .button-container{
                 display: flex;
                 flex-direction: column;
                 justify-content: flex-end;
                 align-items: flex-end;
-                height: 100%;
-                width: 47%;
+                width: 605px;
```

Comment:
So the button container is fixed 605px in smaller devices?

Reply:
yes if the resolution of  X-row  is smaller than 1400px